m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -180,7 +183,7 @@ list of institution and member limits, eligible shares, list of scheduled transf
   "rgState": "CREATETRAN",
   "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
   "userChrList": [
-    { "id": 1, "value": "1234567890S0001|9876543210|L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][recipient member id][account type or id][frequency]|[startDate]|[day1]|[day2]
+    { "id": 1, "value": "1234567890S0001|9876543210|L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][recipient member id][account type or id][frequency]|[startDate || "soonest" ]|[day1]|[day2]
```

Comment:
As it appears you are no longer using Account type in section [account type or id] consider removing the word account type.
so in the contract for CREATETRAN '[account type or id]' Would become [id]. Let me know if I'm mistaken.

Reply:
Jordan, you're correct... this should be changed to "recipient id"